packages/bpk-component-modal/src/BpkModalDialog.js

```diff
@@ -66,17 +66,18 @@ const BpkModalDialog = props => {
             {props.title}
           </h2>
           &nbsp;
-          {props.closeText ? (
-            <BpkButtonLink onClick={props.onClose}>
-              {props.closeText}
-            </BpkButtonLink>
-          ) : (
-            <BpkCloseButton
-              className={getClassName('bpk-modal__close-button')}
-              label={props.closeLabel}
-              onClick={props.onClose}
-            />
-          )}
+          {props.closeOnScrimClick &&
```

Comment:
The behaviour of hiding the close button/link should be a separate prop from `closeOnScrimClick`.

Reply:
Done.